config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -229,3 +229,15 @@ presubmits:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20190420-93fab49-experimental
         command:
         - ./hack/verify-github-spelling.sh
+
+  - name: pull-test-infra-yamllint
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: quay.io/kubermatic/yamllint:0.1
+        command:
+        - yamllint
```

Comment:
Is there an automatic delinter?

Typically we have a hack/update-yaml.sh and hack/verify-yaml.sh scripts to update and validate this sort of thing.

Reply:
Not to my knowledge. There are also some rules for which this wouldn't really be possible, e.G. `duplicate keys`